nettacker/config.py

```diff
@@ -1,14 +1,13 @@
+CWD = Path.cwd()
```

Comment:
This isn't going to work as `Path` isn't defined at this point yet. Please test your code before making a PR.